hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -177,6 +178,7 @@ public static void main(String... args) throws IOException {
                 dumpLoc.write("➡️\n\n");
                 dumpLoc.write(item.itemRecord() + "\n\n");
             }
+            dumpLoc.flush();
```

Comment:
@david-bakin-sl pointed out if we use a `try`-with-resources here we don't need this; upstream branch should work now without the explicit `flush()`.

Reply:
Ah awesome will update it